test/proposals/ListOnZoraProposalForked.t.sol

```diff
@@ -208,36 +165,126 @@ contract ListOnZoraProposalForkedTest is TestUtils {
+        _expectEmit3();
+        emit AuctionEnded(
+            proposalData.token,
+            proposalData.tokenId,
+            IReserveAuctionCoreEth.Auction({
+                seller: address(party),
+                reservePrice: uint96(proposalData.listPrice),
+                sellerFundsRecipient: address(party),
+                highestBid: uint96(proposalData.listPrice),
+                highestBidder: john,
+                duration: uint32(proposalData.duration),
+                startTime: auctionStartTime,
+                firstBidTime: auctionStartTime
+            })
+        );
+        _expectEmit3();
         emit ZoraAuctionSold(proposalData.token, proposalData.tokenId);
-        assertTrue(proposal.executeListOnZora(executeParams).length == 0);
-        assertEq(address(proposal).balance, proposalData.listPrice);
+        assertTrue(executeProposal(proposalId, proposal, progressData).length == 0);
+        assertEq(address(party).balance, proposalData.listPrice);
     }
 
     function testForked_canSettleSuccessfulEndedListing() external onlyForked {
         (
-            IProposalExecutionEngine.ExecuteProposalParams memory executeParams,
+            PartyGovernance.Proposal memory proposal,
             ListOnZoraProposal.ZoraProposalData memory proposalData
-        ) = _createExecutionParams();
-        executeParams.progressData = proposal.executeListOnZora(executeParams);
-        _bidOnListing(proposalData.token, proposalData.tokenId, proposalData.listPrice);
+        ) = _buildZoraProposal();
+        (uint256 proposalId, bytes memory progressData) = proposePassAndExecuteProposal(proposal);
+        uint32 auctionStartTime = uint32(block.timestamp);
+        _bidOnListing(john, proposalData.token, proposalData.tokenId, proposalData.listPrice);
         skip(proposalData.duration);
+        _expectEmit3();
+        emit AuctionEnded(
+            proposalData.token,
+            proposalData.tokenId,
+            IReserveAuctionCoreEth.Auction({
+                seller: address(party),
+                reservePrice: uint96(proposalData.listPrice),
+                sellerFundsRecipient: address(party),
+                highestBid: uint96(proposalData.listPrice),
+                highestBidder: john,
+                duration: uint32(proposalData.duration),
+                startTime: auctionStartTime,
+                firstBidTime: auctionStartTime
+            })
+        );
         ZORA.settleAuction(proposalData.token, proposalData.tokenId);
-        _expectEmit0();
+        _expectEmit3();
         emit ZoraAuctionSold(proposalData.token, proposalData.tokenId);
-        assertTrue(proposal.executeListOnZora(executeParams).length == 0);
-        assertEq(address(proposal).balance, proposalData.listPrice);
+        assertTrue(executeProposal(proposalId, proposal, progressData).length == 0);
+        assertEq(address(party).balance, proposalData.listPrice);
+    }
+
+    /// MARK: Helpers
```

Comment:
Delete this?

Reply:
i kinda like it. If you are against i can delete

Reply:
Not opinionated on it, if you like it we can keep it